src/main.ts

```diff
@@ -215,69 +222,86 @@ class AutoEntities extends LitElement {
       return typeof entity === "string" ? { entity: entity.trim() } : entity;
     };
 
+    let [sort] = await Promise.all([
+      this._config.sort
+        ? get_sorter(this.hass, this._config.sort)
+        : Promise.resolve((x: EntityList) => Promise.resolve(x)),
+    ]);
```

Comment:
This seems a bit unnecessary?
Wouldn't this work?

```js
let sort = this._config.sort
    ? await get_sorter(this.hass, this._config.sort)
    : async (x) => x;
```

I also think `(x) => 0` is more correct for a NOOP sorter?